mantidproject/mantid
# Add algorithm SaveFITS

Adds a new algorithm, SaveFITS, to support other imaging/tomography algorithms and interfaces.

**To test:**
- Code review
- Check the documentation of the algorithm, and the usage examples. It should be easy to understand.
- The unit, docs and system tests should pass and look sensible.
- There's a few fixes in LoadFITS (other issues are open against it for more refactoring and fixes) which are covered by unit and system tests. It's been modified to not load all the headers at once before loading the file contents (this was causing user unfriendly delays without progress feedback when loading large stacks over the network).

Fixes #15944, 
Fixes #13887.

Release notes: https://github.com/mantidproject/mantid/commit/253929fd99b20ee2475f1f3d004b456cda08f254

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
